torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -205,6 +215,48 @@ void InplaceMKLDNNSubgraph(std::shared_ptr<Graph> graph) {
   }
 }
 
+// aten_op is a sequence of aten ops that implement a required operation
+// given input output tensors
+Operation createUnaryOp(
+    std::function<void(at::Tensor output, at::Tensor input)> aten_op,
+    bool inplace = false) {
+  return [aten_op, inplace](Stack* stack) {
+    auto a = pop(stack).toTensor();
+    c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
+    // we cast `a` to an `ideep::tensor`, so we can get at its descriptor
+    // which we then use to set up `out` tensor w/ the same props as a
+    auto a_it = at::native::itensor_from_mkldnn(a);
+    auto raw_data = a_it.get_data_handle();
+    auto topt = a.options().layout(c10::kStrided);
```

Comment:
can you rename `topt` ? I don't know where `t` or `opt` is coming from. Maybe: `a_options_with_strided`